README.md

```diff
@@ -139,6 +141,51 @@ const moduleConfig: OktaConfig = { oktaAuth };
 export class MyAppModule { }
 ```
 
+### `APP_INITIALIZER`
+
+Starting with `okta-angular 6.2.0`, you can provide `OktaConfig` in `APP_INITIALIZER` provider factory with method `setConfig()` of `OktaAuthConfigService` instance.
```

Comment:
Should this include "which allows you to load the Okta config at runtime" or something like that?